### PR TITLE
Feature/alcohol query

### PR DESCRIPTION
**Pull request by PizzamanJ10**
Added 3 functions:
- addAlcoholic
- removeAlcoholic
- getAlcoholics

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,5 @@ how to use:
 
 my setup is ec2 in aws.
 I didn't use Docker, so if you want your project to run on linux (at least ubuntu) you need to install all the packages in the Docker file.
+
+Hi I'm Daniel and I created this branch
```
Comment (itamarshubin) [inline README.md]: WE DON'T CARE ABOUT YOU.
REMOVE THIS SHIT

File: src/firebase/shabbas-manage.ts [hunk 2]
```diff
@@ -46,6 +46,71 @@ export const addUser = async (msg: Message) => {
   }
 };
 
+// Add alcoholic function
```
Comment (itamarshubin) [inline src/firebase/shabbas-manage.ts]: if your code is understandable you don't need this fucking comment

File: src/firebase/shabbas-manage.ts [hunk 3]
```diff
@@ -46,6 +46,71 @@ export const addUser = async (msg: Message) => {
...
+
+  if (shabbas.data().alcoholics?.find((par: { id: string }) => par.id === userRef.id)) {
+    await client.sendMessage(msg.from, "אתה כבר נרשמת לאלכוהול.");
+  } else {
+    await updateDoc(shabbas.ref, { alcoholics: arrayUnion(userRef.ref) });
+    await client.sendMessage(msg.from, "אחלה, אלכוהול זה טוב.");
+  }
+};
+
+// Remove alcoholic function
```
Comment (itamarshubin) [inline src/firebase/shabbas-manage.ts]: Same

File: src/message-handler.ts [hunk 4]
```diff
@@ -65,6 +68,22 @@ export const messageHandler = async (msg: Message) => {
     return;
   }
 
+  //Alcohol Functions
```
Comment (itamarshubin) [inline src/message-handler.ts]: Same

Comment (itamarshubin) [reply src/message-handler.ts]: 🖕

**Review by itamarshubin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PizzamanJ10) [reply README.md]: My therapist told me it's healthy tho

File: src/firebase/shabbas-manage.ts (hunk 2 above)
Comment (PizzamanJ10) [reply src/firebase/shabbas-manage.ts]: It's your code so it isn't understandable and needs the comment

File: src/firebase/shabbas-manage.ts (hunk 3 above)
Comment (PizzamanJ10) [reply src/firebase/shabbas-manage.ts]: same same

File: src/message-handler.ts (hunk 4 above)
Comment (PizzamanJ10) [reply src/message-handler.ts]: 8=======>

**Review by itamarshubin — APPROVED** [review]

**Pull request merged**